components/style/properties.mako.rs

```diff
@@ -621,7 +624,7 @@ pub mod longhands {
 
     </%self:longhand>
 
-    ${single_keyword("position", "static absolute relative fixed")}
+    ${single_keyword("position", "static absolute relative fixed", extra_gecko_values="sticky".split())}
```

Comment:
I think it would be more syntactically consistent to make extra_foo_values a string with space-separated tokens, and move the parsing for it into the Keyword class constructor.

(I also think we should switch the product lists from "gecko,servo" to "gecko servo" at some point - my bad on introducing that wart).